packages/flame_rive/lib/src/rive_component.dart

```diff
@@ -0,0 +1,177 @@
+import 'dart:async';
+import 'dart:math';
+import 'dart:ui' as ui;
+
+import 'package:flame/components.dart';
+import 'package:flutter/rendering.dart';
+import 'package:rive/rive.dart';
+
+// ignore_for_file: implementation_imports
+import 'package:rive/src/rive_core/math/aabb.dart';
+import 'package:rive/src/rive_core/math/mat2d.dart';
+import 'package:rive/src/rive_core/math/vec2d.dart';
+
+class RiveComponent extends PositionComponent {
+  final Artboard artboard;
+  final RiveArtboardRenderer _renderer;
+
+  RiveComponent({
+    required this.artboard,
+    bool antialiasing = true,
+    bool useArtboardSize = true,
+    BoxFit fit = BoxFit.contain,
+    Alignment alignment = Alignment.center,
+
+    // position component arguments
```

Comment:
Do we have this notation/structure somewhere else?

Reply:
Not sure what are you referring to.

Reply:
Having a comment in the middle of the constructor giving a section to a subset of the arguments.